libbeat/processors/convert/convert.go

```diff
@@ -403,3 +403,13 @@ func cloneValue(value interface{}) interface{} {
 		return value
 	}
 }
+
+// Helper to interpret a string as either base-10 or base-16.
```

Comment:
nit: can you use the `strToInt blah blah` godoc format here.

Reply:
done